test/libyul/objectCompiler/subObjectAccess.yul

```diff
@@ -0,0 +1,104 @@
+object "A" {
```

Comment:
Can we have multiple objects on the root level?

Reply:
Should not be possible.

Reply:
I think yul does not allow multiple objects on the root level. It fails in parsing stage with `expected end of source but got identifier`